Mathlib/RingTheory/Localization/FractionRing.lean

```diff
@@ -24,7 +24,7 @@ import Mathlib.RingTheory.Localization.Basic
 
 ## Implementation notes
 
-See `RingTheory/Localization/Basic.lean` for a design overview.
+See `Mathlib/RingTheory/Localization/Basic.lean` for a design overview.
```

Comment:
I think we usually omit `Mathlib/`. Alternatively please change it everywhere (in `RingTheory/Localization`) if you  think this is better.

Reply:
No, omitting `Mathlib/` will just produce a link which links to 404, see https://leanprover-community.github.io/mathlib4_docs/Mathlib/RingTheory/Localization/FractionRing.html

> Alternatively please change it everywhere (in `RingTheory/Localization`) if you think this is better.

Sure, if I spot them.

Reply:
Done.